tests/conftest.py

```diff
@@ -14,13 +14,15 @@
 
 EXAMPLE_DATA = Path(__file__).parents[1] / "example_data"
 
-HARVEST_SOURCE_URL = os.getenv('HARVEST_SOURCE_URL')
+HARVEST_SOURCE_URL = os.getenv("HARVEST_SOURCE_URL")
 
 
 @pytest.fixture(scope="session")
 def app() -> Flask:
     app = create_app()
 
+    app.config["TESTING"] = True
+
```

Comment:
I don't think we need this anymore, do we?

Reply:
removed